docker-compose.yml

```diff
@@ -8,14 +14,14 @@ services:
     networks:
       - common
     environment:
-      # @see https://phabricator.wikimedia.org/source/mediawiki/browse/master/includes/DefaultSettings.php
-      # The following line is replaced by a script in
-      # provision/mariadb/01-databases.sql because we need to initialize 2
-      # databases: my_wiki and eauth.
-      # MYSQL_DATABASE: my_wiki
-      MYSQL_USER: wikiuser
-      MYSQL_PASSWORD: example
+      # Multiple dbs provisioned in provision/mariadb/01-databases.sql
+      # MYSQL_DATABASE: my_wiki 
```

Comment:
The explanatory lines are missing in this PR. Need to explain why we do multiple dbs.

Reply:
I condensed it into one line here to make the file less verbose. I'll expand the full text in the dev manifest though IMO it's not necessary to spell out the details.

Reply:
I had revisited this section of the docker-compose.yml and had forgotten why this line is commented out. It would be even less clear for the user when reading this line.